iml-gui/crate/src/components/tree.rs

```diff
@@ -752,6 +756,60 @@ mod tests {
         )
     }
 
+    #[test]
+    fn test_cache_cloning() {
+        let c1 = fixtures::get_cache();
+        let mut c2: Cache = c1.clone();
+        let mut c3: Cache = c2.clone();
+
+        // The hashmap ost_pool_osts is the same for all cache clones c1, c2, c3
```

Comment:
You are basically testing what must be already tested in std library. Probably there is not value in these tests here.

Reply:
I'd leave it here.
This (among the other things) shows the common usage pattern of the data structure similar to the `Cache` (it took some time for me and I'm not sure I can figure it out easily after a half of a year).

Also it shows the common invariants to expect when working with such data structures.

Reply:
But that's in [the documentation](https://doc.rust-lang.org/std/sync/struct.Arc.html) :)